app/common/src/map/Address.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import Color from '../Color';
+
+type Props = {||};
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    borderTopColor: Color.grey._100,
+    borderTopWidth: 1,
+    paddingVertical: 10,
+    marginHorizontal: 10,
+  },
+  mapIcon: {
+    alignSelf: 'center',
+    marginRight: 10,
+  },
+});
+
+class Address extends React.Component<Props> {
+  render = () => {
+    return (
+      <View style={styles.container}>
+        <View style={styles.mapIcon}>
+          <Ionicons name="md-map" size={24} color={Color.brand} />
+        </View>
+        <View>
+          <Text style={{ fontWeight: 'bold' }}>Address</Text>
+          <Text>950 Candice Wells Suite 375, Dublin</Text>
```

Comment:
I would expect props here instead of text.

Reply:
fixed